### ISSUE TITLE
Change direct image markdown into image links

### ISSUE THREAD

**Issue by ThePsionic** [gitterHQ/irc-bridge#44]
Right now, if a direct image is sent, it will show as `[![blob](thumbURL)](imgURL)]`. In most common IRC clients, this will lead to the thumbURL and the imgURL combining into one huge clickable link, so that the user has to manually copy the last part of the message to see the image. Since IRC users don't have a use for the thumbnail images anyway, it'd be nice if just the image link itself would be sent.

**Comment by cwensley** [comment]
Yes, since `)](` characters are technically valid URL characters, images passed through IRC should not use them.  Or, at least surround the urls with spaces.

**Comment by nikitavoloboev** [comment]
Is it possible to somehow get around this?
